src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -14,6 +14,8 @@ import java.time.Instant
 class ComposeUp extends DefaultTask {
 
     private Map<String, ServiceInfo> servicesInfos = new HashMap<>()
+    private List<String> composeFiles

```

Comment:
I would prefer to work with `extension.useComposeFiles` property directly (so remove this field).